jobfunnel/config/cli.py

```diff
@@ -151,18 +151,28 @@ def parse_cli(args: List[str]) -> Dict[str, Any]:
     search_group.add_argument(
         '-ps',
         dest='search.province_or_state',
-        type=str,
+        type=None,
```

Comment:
Is it possible to make this an `Optional[str]` since we want to make sure this is a string if the user provides a value.